internal/cluster/statemachine.go

```diff
@@ -218,7 +218,8 @@ func NewClusterStateMachine(th *transitionHandler) stateswitch.StateMachine {
 		stateswitch.State(models.ClusterStatusPreparingForInstallation),
 		stateswitch.State(models.ClusterStatusFinalizing),
 		stateswitch.State(models.ClusterStatusInstalled),
-		stateswitch.State(models.ClusterStatusError)} {
+		stateswitch.State(models.ClusterStatusError),
+		stateswitch.State(models.ClusterStatusAddingHosts)} {
```

Comment:
Is there any transition from and to this new state "ClusterStatusAddingHosts" ?

Reply:
Cluster in state in day2 is irrelevant, since all the action are done on the nodes and each node is independent of other nodes. Cluster is just a wrap around nodes so that we can view their state ( as in day1). So we created a new state, and cluster always stays in that state. We also added that state to various "allow" list for updates etc'

Reply:
So why do we need this state at all?

Reply:
i added it in order to use other APIs (create download ISO) update cluster etc', in order not to change code too much